.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Issue template
+about: Define de structure of a task. Help us to be more specific. Avoid the incertitude
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Journey or Title
+For titles use the format is 'As a <ROLE>, i want to <GOAL>, so that <REASON>'
+For journeys  add a little description of what's actually happen and what we expected to happen
```

Comment:
There are two spaces between `journeys` and `add`
I think is should be `what's actually happening` instead of `what's actually happen` or it could also be `what actually happened` or just `what happened`